setup.py

```diff
@@ -21,8 +21,6 @@ def main():
         # not zip-safe.
         zip_safe=False,
         cmdclass=versioneer.get_cmdclass(),
-        tests_require=['six'],
-        test_suite='conda_smithy',
```

Comment:
Yes, we can even use [pytest-runner](https://pypi.python.org/pypi/pytest-runner) if you prefer.